pkg/opensearch/client/client.go

```diff
@@ -313,6 +319,14 @@ func (c *baseClientImpl) executeRequest(method, uriPath, uriQuery string, body [
 		req.SetBasicAuth(c.ds.User, password)
 	}
 
+	
+	if req.Method != http.MethodGet && c.getSettings().Get("serverless").MustBool(false) {
+		h := sha256.New()
```

Comment:
what do you think about using `sha256.Sum256` ( https://pkg.go.dev/crypto/sha256#Sum256) here? seems shorter.

Reply:
fixed in 4be05df